distros/ros2-overlay.nix

```diff
@@ -172,9 +172,11 @@ rosSelf: rosSuper: with rosSelf.lib; {
       # Use X11 by default in RViz2.
       # https://github.com/ros-visualization/rviz/issues/1442
       "--set-default QT_QPA_PLATFORM xcb"
+      "--set QT_QPA_PLATFORM xcb"
     ];
     postFixup = postFixup + ''
       wrapQtApp "$out/lib/rviz2/rviz2"
+      wrapQtApp "$out/bin/rviz2"
```

Comment:
Is this necessary? I see that `bin/rviz2` is already wrapped. I check it this way:
```sh
nix build .#humble.rviz2
strings ./result/bin/rviz2
```
In the output, I see
```
...
# ------------------------------------------------------------------------------------
# The C-code for this binary wrapper has been generated using the following command:
makeCWrapper '/nix/store/dcm483irr16y593c8dx14xzabfbif6na-ros-humble-rviz2-11.2.12-r1/bin/.rviz2-wrapped' \
    --inherit-argv0 \
    --set-default 'QT_QPA_PLATFORM' 'xcb' \
    --prefix 'QT_PLUGIN_PATH' ':' '/nix/store/zp351790kvhsxx197j9d1bsh0vi3vshc-qtbase-5.15.14-bin/lib/qt-5.15.14/plugins' \
    --prefix 'QT_PLUGIN_PATH' ':' '/nix/store/ykzzcnbx2c6rg38a2bqpbpwjmm0wzjlb-qtsvg-5.15.14-bin/lib/qt-5.15.14/plugins' \
    --prefix 'QT_PLUGIN_PATH' ':' '/nix/store/m9b5dry9baqi6w9v8dydin7p84nwxbgy-qtdeclarative-5.15.14-bin/lib/qt-5.15.14/plugins' \
    --prefix 'NIXPKGS_QT5_QML_IMPORT_PATH' ':' '/nix/store/m9b5dry9baqi6w9v8dydin7p84nwxbgy-qtdeclarative-5.15.14-bin/lib/qt-5.15.14/qml' \
    --prefix 'NIXPKGS_QT5_QML_IMPORT_PATH' ':' '/nix/store/fzqynncbhji975j3l12gkbcjdaby4h5c-qtquickcontrols-5.15.14/lib/qt-5.15.14/qml' \
    --prefix 'QT_PLUGIN_PATH' ':' '/nix/store/vzdhbc4c6irp3khmxmmdimli8yri5f5r-qtwayland-5.15.14-bin/lib/qt-5.15.14/plugins' \
    --prefix 'NIXPKGS_QT5_QML_IMPORT_PATH' ':' '/nix/store/vzdhbc4c6irp3khmxmmdimli8yri5f5r-qtwayland-5.15.14-bin/lib/qt-5.15.14/qml' \
    --prefix 'XDG_DATA_DIRS' ':' '/nix/store/dcm483irr16y593c8dx14xzabfbif6na-ros-humble-rviz2-11.2.12-r1/share'
...
```